snow/engine/snowman/bootstrap/storage.go

```diff
@@ -206,8 +210,22 @@ func execute(
 				return err
 			}
 
+			lastProcessedKey := iterator.Key()
```

Comment:
nit: update comment on lines 200/201 to mention that we don't just reduce memory pressure but we also compact?